tests/unit/moduleapi/misc.tcl

```diff
@@ -16,6 +16,14 @@ start_server {tags {"modules"}} {
         assert { [string match "*cmdstat_module*" $info] }
     }
 
+    test {test redis version} {
+        set info [r info server]
+        set temp [lindex [split $info "\n"] 1]
+        set temp1 [lindex [split $temp ":"] 1]
+        set version [string trim $temp1]
```

Comment:
```suggestion
        set version [s version]
```
`s` is a proc that returns an INFO field.

Reply:
Nice, was not aware of it.